Plugins/ExternalTexture/Source/ExternalTexture_D3D11.cpp

```diff
@@ -1,29 +1,38 @@
 #include <Babylon/Graphics/Device.h>
-#include <Babylon/GraphicsInterops/ExternalTexture.h>
+#include <Babylon/Plugins/ExternalTexture/ExternalTexture.h>
 #include <Babylon/Graphics/RendererType.h>
 #include <Babylon/Graphics/Texture.h>
 #include <Babylon/Graphics/DeviceContext.h>
 
 #include <napi/env.h>
 #include <napi/napi_pointer.h>
 
-namespace Babylon::Graphics
+namespace Babylon::Plugins
 {
     //For refrence see renderer_d3d11.cpp line 203.
```

Comment:
Putting a specific line number here may get out of sync. Maybe put a link to a specific commit with a line number?
```suggestion
    // For reference see renderer_d3d11.cpp line 203.
```